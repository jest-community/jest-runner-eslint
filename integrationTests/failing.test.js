const runJest = require('./runJest');

it('Works when it has failing tests', async () => {
  expect(await runJest('failing')).toMatchSnapshot();
});
