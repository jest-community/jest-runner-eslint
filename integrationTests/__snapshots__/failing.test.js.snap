// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has failing tests 1`] = `
"FAIL integrationTests/__fixtures__/failing/__eslint__/file.js
  ✕ ESLint
/mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/failing/__eslint__/file.js
  4:3  error  'hello' is not defined  no-undef
  5:3  error  'bye' is not defined    no-undef
✖ 2 problems (2 errors, 0 warnings)
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.

"
`;
